### DISCUSSION TITLE
Connectivity issue with dexcom g7. Samsung s23 ultra. Android 14

### DISCUSSION THREAD

**Discussion by joshg1994** [NightscoutFoundation/xDrip#3314] [Q&A] [answered]
I'm having connectivity issues, just downloaded xdrip for first time and can't seem to connect my CGM.  It has been over an hour of me waiting
![Screenshot_20240124_182327_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/157648698/e313a544-bf6f-4b25-864c-0d21086a357e)
![Screenshot_20240124_183755_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/157648698/4d3cdefc-d9aa-48a1-9cb7-e149676dc189)

**Comment by Navid200** [comment]
Have you applied the G7 device to your skin?
Did you only try xDrip or did you also try Dexcom app?
Did you get a pair request with xDrip and did you approve?

**Reply by joshg1994** [reply]
I have applied g7 to to my skin, paired with dexcom then deleted the g7 app prior to downloading xDrip. I never got a pair request

**Reply by Navid200** [reply, marked as answer]
Uninstall xDrip.
Keep Dexcom app uninstalled.
Go to your phone Android settings.  Go to connected Bluetooth devices.
Forget all devices named DXCM** where ** could be any two characters.

Install xDrip again.
Choose G4, G5, G6, G7, 1
Tap on xDrip settings.  It will ask you to give permission.  Choose to change it in settings.  In settings, choose "Allow all the time".
Scan the QR code from here: https://navid200.github.io/xDrip/docs/Dexcom/G7.html

Enter the pairing code as the transmitter ID.
Go to the status page and wait for pairing request.
Approve.

It should provide readings in a few minutes.
